atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/snapshot/DefaultKeyValueSnapshotReader.java

```diff
@@ -192,8 +193,10 @@ private <T> ListenableFuture<Collection<Map.Entry<Cell, T>>> getWithPostFilterin
                     while (!remainingResultsToPostFilter.isEmpty()) {
                         remainingResultsToPostFilter = AtlasFutures.getUnchecked(getWithPostFilteringInternal(
                                 tableReference, remainingResultsToPostFilter, resultsAccumulator, transformer));
-                        Preconditions.checkState(
-                                ++iterations < MAX_POST_FILTERING_ITERATIONS,
```

Comment:
this was the primary one that popped up in my analysis

Reply:
fair enough: yeah this would get triggered on every user get